src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -163,6 +163,18 @@ bool RhConfig::ReadKnobBooleanValue(_In_z_ const char *name, bool* pValue)
     return false;
 }
 
+bool RhConfig::ReadKnobDoubleValue(_In_z_ const char *name, double* pValue)
+{
+    const char *embeddedValue = nullptr;
+    if (GetEmbeddedVariable(&g_compilerEmbeddedKnobsBlob, name, false, &embeddedValue))
+    {
+        *pValue = strtod(embeddedValue, NULL);
```

Comment:
- Do we want to support all formats `strtod` supports?
- What if the `embeddedValue` is not a valid floating point number?

Reply:
This will take care of error handling
```suggestion
        char *endptr;
        *pValue = strtod(embeddedValue, &endptr);
        if (embeddedValue == endptr) return false; // invalid input
```

Reply:
This will take care of error handling
```suggestion
        char *endptr;
        double val = strtod(embeddedValue, &endptr);
        if (embeddedValue == endptr) return false; // invalid input; no conversion was performed
        *pValue = val;
```

Reply:
> * Do we want to support all formats `strtod` supports?
> * What if the `embeddedValue` is not a valid floating point number?

Thanks for the review! Do I also need to add error handling for strtoull in `ReadConfigValue` and `ReadKnobUInt64Value`